pkg/controller/migmigration/annotation.go

```diff
@@ -78,6 +78,9 @@ const (
 	StagePodLabel = "migration.openshift.io/is-stage-pod"
 )
 
+// Bucket limit for number of items annotated in one Reconcile
+const NumberOfItems = 50
```

Comment:
Let's name this to something more specific since this is a package level const. 

Perhaps `AnnotationsPerReconcile`